examples/instance/main.go

```diff
@@ -74,15 +74,20 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	if len(*createRespOk) == 0 {
+		log.Fatal("create response is empty")
+	}
 	log.Printf("***************[1]****************** %+v\n", *createRespOk)
 
-	insIDs := make([]string, 0)
+	insID := ""
 	for _, in := range *createRespOk {
-		insID := in.PvmInstanceID
-		insIDs = append(insIDs, *insID)
```

Comment:
We don't need to append the insID in the new version?

Reply:
This example only creates 1 instance hence we can directly use the first instance response from the response list.

Reply:
Ah okay looks good!